quora-api/src/main/java/com/upgrad/quora/api/controller/SignupController.java

```diff
@@ -0,0 +1,42 @@
+package com.upgrad.quora.api.controller;
+
+import com.upgrad.quora.api.model.SignupUserRequest;
+import com.upgrad.quora.api.model.SignupUserResponse;
+import com.upgrad.quora.service.business.SignupBusinessService;
+import com.upgrad.quora.service.entity.UserEntity;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/")
+public class SignupController {
```

Comment:
Rename/refector "SignupController" to "UserController". It is mentioned in "REST API endpoints - 1". Please follow the naming convention.

Reply:
"SignupController" name changed to "UserController".